Buildings/Controls/OBC/CDL/Conversions/BooleanToReal.mo

```diff
@@ -22,16 +22,16 @@ equation
     Documentation(
       info="<html>
 <p>
-Block that outputs the <code>Boolean</code>
-equivalent of the <code>Integer</code> input.
+Block that outputs the <code>Real</code>
+equivalent of the <code>Boolean</code> input.
 </p>
 <pre>
-  y = if u then integerTrue else integerFalse;
+  y = if u then realTrue else realFalse;
 </pre>
 <p>
 where <code>u</code> is of <code>Boolean</code> and <code>y</code>
-of <code>Integer</code> type,
-and <code>integerTrue</code> and <code>integerFalse</code> are parameters.
+of <code>Real</code> type,
+and <code>realTrue</code> and <code>realFalse</code> are parameters.
 </p>
 </html>",
       revisions="<html>
```

Comment:
Please also add this in the revision notes of this model, and in the revision notes of `Buildings/package.mo`.

Reply:
Revision and release notes added in 8065511.